UI: use newline to separate sentences in popup message

quacksouls opened the pull request:
From Venusaur (`후시기바나#0028`) on the Bitburner server of Discord:

> One more presentation layer issue
>
> Bitburner v2.1.0 (6ac04717) 
![break](https://user-images.githubusercontent.com/66396308/199869723-a0fe510d-f603-4f30-a8fd-5e26166e7c66.png)

After the player has completed the creation of a program, a popup message appears to inform that the program is completed.  ~We should use whitespace to separate the sentences in the popup message, as done in these files:~

1. ~[src/DarkWeb/DarkWeb.tsx](https://github.com/bitburner-official/bitburner-src/blob/6ac04717fe363b2054bb8b7b28211f9e6f2294e9/src/DarkWeb/DarkWeb.tsx#L83)~
2. ~[src/NetscriptFunctions/Singularity.ts](https://github.com/bitburner-official/bitburner-src/blob/6ac04717fe363b2054bb8b7b28211f9e6f2294e9/src/NetscriptFunctions/Singularity.ts#L546)~

Before:
![break-before](https://user-images.githubusercontent.com/66396308/199870554-6bbeb544-6be4-47ef-bc91-7f0e65f3a6ba.png)

~After:~
![break-after](https://user-images.githubusercontent.com/66396308/199870567-2838782f-b357-4dc0-83ef-22622ef81cbb.png)

Snarling commented:
Should be `\n` instead of a space I think. Preserves how the dialog originally looked and seems like it would be more readable.

quacksouls committed:
UI: use newline to separate sentences in popup message

After the player has completed the creation of a program, a popup message appears to inform that the program is completed.  Use the newline character, instead of the HTML tag <br>, to separate the sentences in the popup message.

quacksouls commented:
> Should be `\n` instead of a space I think. Preserves how the dialog originally looked and seems like it would be more readable.

Good catch!  The commit is updated.  Here's what the dialog box looks like when a newline character is used for line break.

![program](https://user-images.githubusercontent.com/66396308/200083288-7cbaafd8-a8b8-48d0-bdb7-028ddfb76af5.png)
